cmd.go

```diff
@@ -236,6 +237,12 @@ func (c *CmdClause) Command(name, help string) *CmdClause {
 	return cmd
 }
 
+// SubcommandOptional makes subcommands optional
+func (c *CmdClause) SubcommandOptional() *CmdClause {
```

Comment:
Could you name this `OptionalSubcommands()`? It reads better in the fluent style.